test/i18n.spec.ts

```diff
@@ -14,6 +14,7 @@ import { I18n } from '../src/I18n'
 import { setup, fs } from '../test-helpers'
 import { I18nManager } from '../src/I18nManager'
 import { validatorBindings } from '../src/Bindings/Validator'
+import { json } from 'mrm-core'
```

Comment:
Are we using this helper somewhere in the test?

Reply:
Note that if we keep this helper, it should go before the relative imports.

Reply:
Seems like my IDE imported it when I did `JSON.stringify`, I have removed it in the latest commit